src/libraries/System.Security.Cryptography.X509Certificates/tests/CertTests.cs

```diff
@@ -254,6 +282,21 @@ public static void X509Cert2CreateFromPfxWithPassword()
             }
         }
 
+        [Fact]
+        public static void X509Cert2CreateFromPfxWithSpanPassword()
+        {
+            Span<char> pw = stackalloc char[4];
```

Comment:
Simpler:
```cs
Span<char> pw = stackalloc char[] { 't', 'e', 's', 't' };
```

Reply:
Ooh, nice.  I didn't know array initializers were allowed in stackalloc expressions.